babble/build.gradle

```diff
@@ -30,13 +30,13 @@ apply plugin: io.mosaicnetworks.getbabblecore.GetBabbleCorePlugin
 //##################################################################################################
 // babble core configuration
 
-def babbleCoreVersion = "0.8.0"
-def repository = "https://github.com/mosaicnetworks/babble/releases/download"
-def method = "https"
+// def babbleCoreVersion = "0.8.0"
+// def repository = "https://github.com/mosaicnetworks/babble/releases/download"
+// def method = "https"
 
-// def babbleCoreVersion = "develop_c149a5051ac7769d927c0374157b3169e8751c10"
-// def repository = (System.getenv('GOPATH') == null ? System.properties['user.home'] + "/go" : System.getenv('GOPATH'))  + "/src/github.com/mosaicnetworks/babble/build/distmobile"
-// def method = "file"
+def babbleCoreVersion = "develop_e42d59028ed5465209d54d6dbc2f238042559f37"
+def repository = (System.getenv('GOPATH') == null ? System.properties['user.home'] + "/go" : System.getenv('GOPATH'))  + "/src/github.com/mosaicnetworks/babble/build/distmobile"
+def method = "file"
 
 //##################################################################################################
```

Comment:
Should probably be a proper babble release before it is promoted to babble.

Reply:
Agreed, we can do that when we merge from develop to master